airflow/hooks/dbapi_hook.py

```diff
@@ -38,7 +38,7 @@ class DbApiHook(BaseHook):
     # Override if this db supports autocommit.
     supports_autocommit = False
     # Override with the object that exposes the connect method
-    connector = None
+    connector = None  # type: None
```

Comment:
Can you enter a protocol here? This variable is not always equal to None, so entering this annotation will be misleading.